build.sbt

```diff
@@ -38,7 +37,8 @@ libraryDependencies ++= Seq(
   libraryDependencies ++= Seq(
     "org.apache.spark" % "spark-core_2.10" % sparkVersion excludeAll(excludeHadoop),
     "org.apache.spark" % "spark-mllib_2.10" % sparkVersion excludeAll(excludeHadoop),
-    "org.apache.spark" % "spark-sql_2.10" % sparkVersion excludeAll(excludeHadoop)
+    "org.apache.spark" % "spark-sql_2.10" % sparkVersion excludeAll(excludeHadoop),
+    "edu.berkeley.cs.amplab" % "mlmatrix_2.10" % "0.1" excludeAll(excludeHadoop)
   )
```

Comment:
Breeze is another one which is a shared dependency between this project and ml-matrix (See https://github.com/amplab/ml-matrix/blob/master/build.sbt) -- It might be fine to not have Breeze though

cc @etrain